lib/index.js

```diff
@@ -91,7 +91,7 @@ module.exports = function nuts(opts) {
                 if (req.useragent.isLinux64) platform = platforms.LINUX_64;
             }
 
-            if (!platform) return next(new Error('No platform specified and impossible to detect one'));
+            if (!platform) return next(new Error('We could not detect your platform.\n\nPlease pick a download directly from: https://github.com/itchio/itch/releases\n\nThanks!'));
```

Comment:
Ha, I didn't expect the PR to be based on master. Will fix later today, ignore for now